compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -1629,16 +1630,22 @@ options numProcessors =
         <> cmdClean
       )
 
-parserInfo :: Int -> ParserInfo Command
-parserInfo numProcessors =
-  info (helper <*> options numProcessors)
-    (  fullDesc
-    <> progDesc "Invoke the Daml compiler. Use -h for help."
-    <> headerDoc (Just $ PP.vcat
-        [ "damlc - Compiler and IDE backend for the Daml programming language"
-        , buildInfo
-        ])
-    )
+parserInfo :: Int -> Bool -> ParserInfo Command
+parserInfo numProcessors addBuildArgsBackupParser =
+  backupParserWithBuildArgs numProcessors addBuildArgsBackupParser $
```

Comment:
Can we inline this directly to the call to `options` instead of inspecting the `infoParser` field?